tests/components/nina/test_binary_sensor.py

```diff
@@ -0,0 +1,200 @@
+"""Test the Nina binary sensor."""
+from typing import Any, Dict
+from unittest.mock import patch
+
+from pynina import ApiError
+import pytest
+
+from homeassistant.components.binary_sensor import DEVICE_CLASS_SAFETY
+from homeassistant.components.nina.const import (
+    ATTR_EXPIRES,
+    ATTR_HEADLINE,
+    ATTR_ID,
+    ATTR_SENT,
+    ATTR_START,
+    DOMAIN,
+)
+from homeassistant.config_entries import ConfigEntryState
+from homeassistant.const import STATE_OFF, STATE_ON
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers import entity_registry as er
+
+from tests.common import MockConfigEntry
+
+ENTRY_DATA: Dict[str, Any] = {
+    "slots": 5,
+    "corona_filter": True,
+    "regions": {"083350000000": "Aach, Stadt"},
+}
+
+
+@pytest.fixture
+def dummy_response() -> Dict[str, Any]:
```

Comment:
It would be better to move this into it's own fixture file and use the `load_fixture` helper instead. There are various examples in the existing code that do that.